site/gatsby-site/src/components/discover/SORTING_LISTS.js

```diff
@@ -1,93 +1,86 @@
-    value_fr: 'instant_search-fr_epoch_incident_date_desc',
-    value_ja: 'instant_search-ja_epoch_incident_date_desc',
     label: 'Newest Incident Date',
     faIcon: faCalendarAlt,
     faClasses: 'far fa-calendar-alt',
     type: 'date',
+    instantKey: 'epoch_incident_date_desc',
   },
   {
     name: 'incident-date-asc',
-    value: 'instant_search-en_epoch_incident_date_asc',
-    value_en: 'instant_search-en_epoch_incident_date_asc',
-    value_es: 'instant_search-es_epoch_incident_date_asc',
-    value_fr: 'instant_search-fr_epoch_incident_date_asc',
-    value_ja: 'instant_search-ja_epoch_incident_date_asc',
     label: 'Oldest Incident Date',
     faIcon: faCalendarAlt,
     faClasses: 'far fa-calendar-alt',
     type: 'date',
     division: true,
+    instantKey: 'epoch_incident_date_asc',
   },
   {
     name: 'published-date-desc',
-    value: 'instant_search-en_epoch_date_published_desc',
-    value_en: 'instant_search-en_epoch_date_published_desc',
-    value_es: 'instant_search-es_epoch_date_published_desc',
-    value_fr: 'instant_search-fr_epoch_date_published_desc',
-    value_ja: 'instant_search-ja_epoch_date_published_desc',
     label: 'Newest Published Date',
     faIcon: faCalendarAlt,
     faClasses: 'far fa-calendar-alt',
     type: 'date',
+    instantKey: 'epoch_date_published_desc',
   },
   {
     name: 'published-date-asc',
-    value: 'instant_search-en_epoch_date_published_asc',
-    value_en: 'instant_search-en_epoch_date_published_asc',
-    value_es: 'instant_search-es_epoch_date_published_asc',
-    value_fr: 'instant_search-fr_epoch_date_published_asc',
-    value_ja: 'instant_search-ja_epoch_date_published_asc',
     label: 'Oldest Published Date',
     faIcon: faCalendarAlt,
     faClasses: 'far fa-calendar-alt',
     type: 'date',
     division: true,
+    instantKey: 'epoch_date_published_asc',
   },
   {
     name: 'submitted-date-desc',
-    value: 'instant_search-en_epoch_date_submitted_desc',
-    value_en: 'instant_search-en_epoch_date_submitted_desc',
-    value_es: 'instant_search-es_epoch_date_submitted_desc',
-    value_fr: 'instant_search-fr_epoch_date_submitted_desc',
-    value_ja: 'instant_search-ja_epoch_date_submitted_desc',
     label: 'Newest Submitted Date',
     faIcon: faCalendarAlt,
     faClasses: 'far fa-calendar-alt',
     type: 'date',
+    instantKey: 'epoch_date_submitted_desc',
   },
   {
     name: 'submitted-date-asc',
-    value: 'instant_search-en_epoch_date_submitted_asc',
-    value_en: 'instant_search-en_epoch_date_submitted_asc',
-    value_es: 'instant_search-es_epoch_date_submitted_asc',
-    value_fr: 'instant_search-fr_epoch_date_submitted_asc',
-    value_ja: 'instant_search-ja_epoch_date_submitted_asc',
     label: 'Oldest Submitted Date',
     faIcon: faCalendarAlt,
     faClasses: 'far fa-calendar-alt',
     type: 'date',
+    instantKey: 'epoch_date_submitted_asc',
   },
 ];
 
+const SORTING_LIST = [];
+
+BASE_SORTING_LIST.forEach((item) => {
```

Comment:
nit. Maybe a `.map` is easier to understand because it seems you are going from  `BASE_SORTINF_LIST` to  `SORTING_LIST`?

Reply:
done